third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -282,7 +282,9 @@ public XDebuggerEditorsProvider getEditorsProvider() {
   }
 
   public boolean isRemoteDebug() {
-    return myRemoteDebug;
+    // TODO(devoncarew): This is a hack for now - we want to instead look for prefix matches
+    // if remote debugging, or running a Flutter app.
```

Comment:
Absolutely! Will add some docs. I think we'll iterate on this implementation a few times - would like to not have to hardcode this setting here.